fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -180,6 +180,13 @@ def reset
         `xcrun simctl erase #{self.udid}`
         return
       end
+
+      def delete
+        UI.message("Deleting #{self}")
```

Comment:
This is not printing, do not know why 😕

Reply:
Can you try `UI.important("Deleting #{self}")`?

Reply:
I was not unpacking the ios_versions... silly me 👍